torchao/experimental/tests/test_quant_passes.py

```diff
@@ -9,48 +9,68 @@
 import torch
 from torch.testing import FileCheck
 
-from torchao.experimental.q_dq_layout import QDQLayout
-from torchao.experimental.quant_api import (
-    Int8DynamicActivationIntxWeightConfig,
-)
+from torchao.dtypes import QDQLayout
 from torchao.experimental.quant_passes import (
     replace_q_dq_patterns_with_quantized_linear_ops_pass,
 )
-from torchao.quantization.granularity import PerGroup, PerRow
-from torchao.quantization.quant_api import quantize_
+from torchao.quantization.granularity import PerAxis, PerGroup
+from torchao.quantization.quant_api import (
+    Int8DynamicActivationIntxWeightConfig,
+    MappingType,
+    ZeroPointDomain,
+    quantize_,
+)
 
 
 class TestQuantPasses(unittest.TestCase):
     def test_replace_q_dq_patterns_with_quantized_linear_ops_pass(self):
         layers = []
         layer_to_weight_dtype = {}
-        layer_to_has_weight_zeros = {}
+        layer_to_weight_mapping_type = {}
+        layer_to_weight_zero_point_domain = {}
+        layer_to_weight_granularity = {}
         for weight_dtype in [getattr(torch, f"int{i}") for i in range(1, 9)]:
-            for has_weight_zeros in [True, False]:
-                for has_bias in [True, False]:
-                    idx = len(layers)
-                    layer_to_weight_dtype[idx] = weight_dtype
-                    layer_to_has_weight_zeros[idx] = has_weight_zeros
-                    layers.append(torch.nn.Linear(64, 64, bias=has_bias))
-        activations = torch.randn(2, 1, 64, dtype=torch.float32)
+            for weight_mapping_type in [MappingType.ASYMMETRIC, MappingType.SYMMETRIC]:
```

Comment:
you can use https://github.com/pytorch/ao/blob/9516764a97147231c72377bc1067c5e997de02f5/torchao/prototype/float8nocompile/benchmark/benchmark.py#L74 to reduce indentation